RUFAS/output_manager.py

```diff
@@ -1002,3 +1002,28 @@ def flush_pools(self) -> None:
         self.warnings_pool: Dict[str, OutputManager.pool_element_type] = {}
         self.errors_pool: Dict[str, OutputManager.pool_element_type] = {}
         self.logs_pool: Dict[str, OutputManager.pool_element_type] = {}
+
+    def reload_pool(self, file_path: Path) -> None:
+        """Reloads the Output Manager variables pool from file path provided by user in gnu arg.
+
+        Parameters
+        ----------
+        file_path : str
+            The path to the file to be loaded to the variables pool.
+
+        Raises
+        ------
+        Exception
+            If an error occurs while opening or reading the user-provided file path.
+        """
+        info_map = {"class": self.__class__.__name__,
+                    "function": self.reload_pool.__name__,
+                    }
+        self.add_log("open_json_file", f"Attempting to open {str(file_path)}.", info_map)
+        try:
+            with open(file_path) as file:
+                self.variables_pool = json.load(file)
+                self.add_log("load_data_successful", f"Successfully loaded data from {str(file_path)}.",
+                             info_map)
+        except Exception as e:
+            raise e
```

Comment:
1. when you are not doing anything with the exception and you just re-raise it, you don't need to make an object for it.
2. `reload_var_pool` in main.py does not have a try-catch block. Raising an exception here makes the program crash without letting the user know why that happens. Instead of raising, add an error to the errors pool.
3. Avoid generic exceptions as much as possible. The exceptions that `open` and `json.load` can raise is documented, catch them specifically. 
```suggestion
        except Exception:
            raise
```

Reply:
I will add more specific exceptions for open and json.load.